### PR TITLE
OGW topo info data: switch to interp

### PR DESCRIPTION
**Pull request by jiahe23**
This PR changes TempestRemap to interpolation when preparing orographic information for OGW. 

This 
- largely simplifies the cache calculation;
- makes topo related data calculations consistent between Earth Warp set up and OGW cache;
- removes the (unnecessary) helper files in examples/hybrid.

### CODE REVIEW

File: test/orographic_gravity_wave/ogwd_3d.jl [hunk 1]
```diff
@@ -15,8 +15,7 @@ using ClimaCorePlots
 
 const FT = Float64
 
-include("../../examples/hybrid/orographic_gravity_wave_helper.jl")
-include("../../post_processing/remap/remap_helpers.jl")
+iinclude("../../post_processing/remap/remap_helpers.jl")
```
Comment (szy21) [inline test/orographic_gravity_wave/ogwd_3d.jl:18]:
```suggestion
include("../../post_processing/remap/remap_helpers.jl")
```

**Review by szy21 — APPROVED** [review]

File: src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl [hunk 2]
```diff
@@ -0,0 +1,59 @@
+using NCDatasets
+using Interpolations
+
+function get_OGW_info(Y, orographic_info_rll)
+    FT = Spaces.undertype(axes(Y.c))
+
+    lon, lat, topo_ll = get_topo_ll(orographic_info_rll)
+
+    varnames = ["hmax", "hmin", "t11", "t12", "t21", "t22"]
```
Comment (charleskawczynski) [inline src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl:9]:
```suggestion
```

File: src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            t22 = FT(0),
+            hmin = FT(0),
+            hmax = FT(0),
+        ),
+    )
+
+    cg_lat = Fields.level(Fields.coordinate_field(Y.c).lat, 1)
+    cg_lon = Fields.level(Fields.coordinate_field(Y.c).long, 1)
+
+    for varname in varnames
```
Comment (charleskawczynski) [inline src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl:25]:
```suggestion
    for varname in (:hmax, :hmin, :t11, :t12, :t21, :t22)
```

File: src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        li_obj = linear_interpolation(
+            (lon, lat),
+            getproperty(topo_ll, Symbol(varname)),
```
Comment (charleskawczynski) [inline src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl:28]:
```suggestion
            getproperty(topo_ll, varname),
```

File: src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+            extrapolation_bc = (Periodic(), Flat()),
+        )
+        Fields.bycolumn(axes(Y.c.ρ)) do colidx
+            parent(getproperty(topo_cg, Symbol(varname))[colidx]) .=
+                FT.(li_obj(parent(cg_lon[colidx]), parent(cg_lat[colidx])))
```
Comment (charleskawczynski) [inline src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl:33]:
```suggestion
            parent(getproperty(topo_cg, varname)[colidx]) .=
                FT.(li_obj(parent(cg_lon[colidx]), parent(cg_lat[colidx])))
```

File: src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl (hunk 4 above)
Comment (jiahe23) [reply src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl:28]: varname here is string, I tried a simple test in REPL below
```
julia> a = (;b =1)
(b = 1,)

julia> getproperty(a, "b")
ERROR: MethodError: no method matching getproperty(::NamedTuple{(:b,), Tuple{Int64}}, ::String)
Closest candidates are:
  getproperty(::Any, ::Symbol) at Base.jl:38
  getproperty(::Any, ::Symbol, ::Symbol) at Base.jl:50
Stacktrace:
 [1] top-level scope
   @ REPL[2]:1

julia> getproperty(a, Symbol("b"))
1
```

**Pull request merged**